webui/tslint.json

```diff
@@ -61,6 +61,7 @@
     "no-redundant-jsdoc": true,
     "no-switch-case-fall-through": true,
     "no-var-requires": false,
+    "no-unused-variable": true,
```

Comment:
Why is this necessary?

Reply:
so it actually warning on unused variable and imports. cleaning up unused import allows cleaning unused packages as well.

Reply:
oh sorry, I got confused and thought that this line disabled the check, but it is actually enabling it. Please disregard.